share/doc/homebrew/brew.1.html

```diff
@@ -200,7 +203,7 @@ <h2 id="COMMANDS">COMMANDS</h2>
 source even if a bottle is provided. Dependencies will still be installed
 from bottles if they are available.</p>
 
-<p>If HOMEBREW_BUILD_FROM_SOURCE is set, regardless of whether <code>--build-from-source</code> was
+<p>If <code>HOMEBREW_BUILD_FROM_SOURCE</code> is set, regardless of whether <code>--build-from-source</code> was
```

Comment:
I see spurious changes. Did someone forget to re-run `brew man` when making some final adjustments in #205? :wink: I fixed that for you in 815edc06862fc4c01ca6ac09a5a3a0b36cfcaa24. They will go away if you rebase this PR.